doc/get_started/linux-eve-raspberry-c.md

```diff
@@ -0,0 +1,137 @@
+---
+platform: linux
+device: eve raspberry
+language: c
+---
+
+Run a simple C sample on Eve Raspberry device running Linux
+===
+---
+
+# Table of Contents
+
+-   [Introduction](#Introduction)
+-   [Step 1: Prerequisites](#Prerequisites)
+-   [Step 2: Prepare your Device](#PrepareDevice)
+-   [Step 3: Build and Run the Sample](#Build)
+-   [Tips](#tips)
+
+<a name="Introduction"></a>
+# Introduction
+
+**About this document**
+
+This document describes how to connect **Eve Raspberry** device running **Linux** with Azure IoT SDK. This multi-step process includes:
+-   Configuring Azure IoT Hub
+-   Registering your IoT device
+-   Build and deploy Azure IoT SDK on device
+
+<a name="Prerequisites"></a>
+# Step 1: Prerequisites
+
+You should have the following items ready before beginning the process:
+
+-   [Prepare your development environment][setup-devbox-linux]
+-   [Setup your IoT hub][lnk-setup-iot-hub]
+-   [Provision your device and get its credentials][lnk-manage-iot-hub]
+
+<a name="PrepareDevice"></a>
+# Step 2: Prepare your Device
+-   Follow the instructions from [device website](http://www.ilevia.com/overview/).
+
+<a name="Build"></a>
+# Step 3: Build and Run the sample
+
+<a name="Load"></a>
+## 3.1 Build SDK and sample
+
+-   Open a PuTTY session and connect to the device.
+
+-   Install the prerequisite packages for the Microsoft Azure IoT Device SDK for C by issuing the following commands from the command line on your board:
+
+        sudo apt-get update
+
+        sudo apt-get install -y curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git unzip openjdk-7-jre
```

Comment:
No need to have uuid-dev, uuid, unzip and openjdk anymore. Additionally you can consider using here build-essential rather than gcc and make.